htdocs/install/mysql/migration/14.0.0-15.0.0.sql

```diff
@@ -510,6 +510,20 @@ INSERT INTO llx_c_action_trigger (code,label,description,elementtype,rang) value
 -- VMYSQL4.3 ALTER TABLE llx_user MODIFY COLUMN fk_soc integer NULL;
 -- VPGSQL8.2 ALTER TABLE llx_user ALTER COLUMN fk_soc DROP NOT NULL;
 
+-- In migration 3.2.0 to 3.3.0 there is a element_tag table creation that is not in create table file
+DROP TABLE IF EXISTS llx_element_tag;
```

Comment:
We should never drop a table into a migration, except if table is no more used into code. This means destruction loss of data for user.
Simple creation is enough, the migration wizard already ignore the error if table already exists.

Reply:
What do with people that have dolibarr since 3.2 with the table created from migration 3.2 to 3.3 but with a different structures than the one created here? 
Your right, the script is always executed on migration so drip isn't a good idea here anyway

Reply:
> What do with people that have dolibarr since 3.2 with the table created from migration 3.2 to 3.3 but with a different structures than the one created here? 
In such a case, we must migrate by adding the field we need.